aztec-up/README.md

```diff
@@ -0,0 +1,30 @@
+# The Aztec Installation Script
+
+```
+bash -c "$(curl https://up.aztec.network)"
+```
+
+That is all.
+
+This will install into `~/.aztec/bin` a collection of scripts to help running aztec containers, and will update
+a users `PATH` variable in their shell startup script so they can be found.
+
+- `aztec` - The infrastructure container.
+- `aztec-cli` - A command line tool for interacting with infrastructure.
+- `aztec-nargo` - A build of `nargo` from `noir` that is guaranteed to be version aligned. Provides compiler, lsp and more.
```

Comment:
Maybe it should mention `aztec-sandbox` too in this list?

Reply:
fixed!